### PR TITLE
email PR

### PR DESCRIPTION
**Pull request by DeepPatel-Tatvasoft**

### CODE REVIEW

**Review by Trusha-Tatvasoft — APPROVED** [review]

File: src/app/modules/global-configuration/email/service/email.service.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+@Injectable({
+  providedIn: 'root',
+})
+export class EmailService {
+  constructor(private http: HttpClient) {}
+
+  getEmails(cultureId: number): Observable<ResponseModel<Email[]>> {
+    return this.http.get<ResponseModel<Email[]>>(
+      environment.baseURL + 'GlobalConfigurationEmails/' + cultureId,
```
Comment (divyapalTatva) [inline src/app/modules/global-configuration/email/service/email.service.ts:17]: use backticks

File: src/app/modules/global-configuration/email/service/email.service.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    );
+  }
+
+  saveEmails(emails: Email[]) {
+    return this.http.post<ResponseModel<Email[]>>(
+      environment.baseURL + 'GlobalConfigurationEmails/create',
```
Comment (divyapalTatva) [inline src/app/modules/global-configuration/email/service/email.service.ts:23]: use backticks

File: src/app/modules/global-configuration/email/service/email.service.ts (hunk 1 above)
Comment (DeepPatel-Tatvasoft) [reply src/app/modules/global-configuration/email/service/email.service.ts]: Resolved

File: src/app/modules/global-configuration/email/service/email.service.ts (hunk 2 above)
Comment (DeepPatel-Tatvasoft) [reply src/app/modules/global-configuration/email/service/email.service.ts]: Resolved

**Review by divyapalTatva — APPROVED** [review]

**Pull request merged**